### PR TITLE
Move buffer note higher

### PR DESCRIPTION
**Pull request by RishhiB**
## Description

> move note to all recipes in FF doc to install `buffer` dependency when faced with error

### CODE REVIEW

File: docs/content/docs/recipes/angular.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ This tutorial assumes that you are familiar with the [Fluid Framework Overview](
 
 {{< /callout >}}
 
+{{< callout note >}}
+
+You may need to install an additional dependency to make this demo compatible with Webpack 5. If you receive a compilation error related to a "buffer" package, please run `npm install -D buffer` and try again. This will be resolved in a future release of Fluid Framework.
```
Comment (curtisman) [inline docs/content/docs/recipes/angular.md:29]: Long term, should our package add a dependency to 'buffer' directly, so the user doesn't have to?

Comment (RishhiB) [reply docs/content/docs/recipes/angular.md:29]: Yes, that is the goal. The issue is buffer is such a common transitive dependency that we want to add it to specific packages that use it and not to root

**Comment by scottn12** [comment]
Could we move the callout note to the same spot where we install the fluid-framework package?

**Review by scottn12 — APPROVED** [review]
LGTM!

**Pull request merged**